coqpp/coqpp_parser.ml

```diff
@@ -0,0 +1,34 @@
+open Lexing
```

Comment:
Side remark but you should include the header (`dev/header.ml`) if this is a new file.

Reply:
Yup.

Tangentially, I'd like to eventually write some unit tests against the code.  Not sure if tool code is automatically accessible in the unit test code.

Reply:
This file is still missing a copyright header.

Reply:
Wait for me to push.